templates/index.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<!-- MAIN START -->
+<div class="main-content">
+  <div class="container-fluid">
+    <div class="main-dashboard">
+      <div class="main-content-header">
+        <h3>My invoices</h3>
+        <p>
+          {{ invoices.count }} invoices total with 
+          {{ dues }} on due and 
+          {{ drafts }} drafts
+        </p>
+      </div>
+      <div class="list-invoice-client table-responsive">
+        <table class="table table-hover">
+          <thead>
+            <tr>
+              <th>
+                <a href="#" class="d-flex align-items-center">
+                  Invoice number
+                </a>
+              </th>
+              <th>
+                <a href="#" class="d-flex align-items-center">
+                  Due date
+                </a>
+              </th>
+              <th>
+                <a href="#" class="d-flex align-items-center">
+                  Payment Status
+                </a>
+              </th>
+              <th>
+                <a href="#" class="d-flex align-items-center">
+                  Amount
+                </a>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {% if invoices %}
```

Comment:
READ THE DOCUMENTATION. https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#for-empty